### DISCUSSION TITLE
.SQL files not appearing in SharePoint search

### DISCUSSION THREAD

**Discussion by calebmil** [microsoft-search/pnp-modern-search#4057] [Q&A]
I feel like I should know the answer to this already but it has been a long time.  I have a group of developers that are storing files of type *.SQL in a document library.  I can get results back from Microsoft Search but they are not being included in SharePoint search results.  Is there a setting I can add to my search settings to make the indexer include SQL files in my query results?

**Comment by kasperbolarsen** [comment]
Hi, can you reproduce this behavior on another tenant? I tried it and the sql files showed up as any other kind of file

**Comment by wobba** [comment]
They should be included. If you add a query of `FileType:sql` you can limit to only sql files. My guess is depending on recall volume that sql files are not ranked very high - if you have default ordering of results on rank.

**Comment by calebmil** [comment]
Ultimately, this turned out to be a query parameters issue.  We were trying to specify a particular folder in a library using Path and DefaultEncodingURL managed properties which apparently no longer consistently contain the folder path within the document library.

**Comment by kasperbolarsen** [comment]
Path and Folders doesn't work well together, as far as I recall I normally use DocumentLink for that
